src/Migration/Migration_Scheduler.php

```diff
@@ -45,6 +45,13 @@ public function mark_complete() {
 		do_action( 'action_scheduler/custom_tables/migration_complete' );
 	}
 
+	/**
+	 * @return bool Whether to do background migrations, defaults to true until the migration is complete
+	 */
+	public function do_background_migration() {
+		return apply_filters( 'action_scheduler/custom_tables/do_background_migration', ! $this->is_migration_complete() );
```

Comment:
I think we should type-cast this to `bool` so that we can use identical comparison in the main `Plugin.php` file. (see comments in that file)